packages/grid/_modules_/grid/components/GridViewport.tsx

```diff
@@ -60,6 +60,7 @@ export const GridViewport: ViewportType = React.forwardRef<HTMLDivElement, {}>(
           <GridRowCells
             columns={visibleColumns}
             row={r}
+            id={r.id}
```

Comment:
Would that one change with the next PR?

Reply:
Yes, the next PR changes to get the id from the Map.